src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -117,13 +133,190 @@ const subMessageStyles = css`
 	margin: 5px 0;
 `;
 
-export const ReaderRevenueLinks: React.FC<Props> = ({
+interface Cta {
+	url: string;
+	text: string;
+}
+interface SupportHeaderProps {
+	content: {
+		heading: string;
+		subheading: string;
+		primaryCta?: Cta;
+		secondaryCta?: Cta;
+	};
+	tracking: TestMeta;
+}
+interface SupportHeaderData {
+	module: {
+		url: string;
+		name: string;
+		props: SupportHeaderProps;
+	};
+	meta: TestMeta;
+}
+
+const ReaderRevenueLinksRemote: React.FC<{
+	edition: Edition;
+	countryCode?: string;
+	pageViewId: string;
+	contributionsServiceUrl: string;
+	ophanRecord: OphanRecordFunction;
+}> = ({
+	edition,
+	countryCode,
+	pageViewId,
+	contributionsServiceUrl,
+	ophanRecord,
+}) => {
+	const [
+		supportHeaderResponse,
+		setSupportHeaderResponse,
+	] = useState<SupportHeaderData | null>(null);
+	const [
+		SupportHeader,
+		setSupportHeader,
+	] = useState<React.FC<SupportHeaderProps> | null>(null);
+
+	const [hasBeenSeen, setNode] = useHasBeenSeen({
+		threshold: 0,
+		debounce: true,
+	});
+
+	useEffect(() => {
+		if (hasBeenSeen && supportHeaderResponse) {
+			sendOphanComponentEvent(
+				'VIEW',
+				supportHeaderResponse.meta,
+				ophanRecord,
+			);
+		}
+	}, [hasBeenSeen, supportHeaderResponse, ophanRecord]);
+
+	useEffect((): void => {
+		setAutomat();
+
+		const requestData = {
+			tracking: {
+				ophanPageId: pageViewId,
+				platformId: 'GUARDIAN_WEB',
+				referrerUrl: window.location.origin + window.location.pathname,
+				clientName: 'dcr',
+			},
+			targeting: {
+				showSupportMessaging: !shouldHideSupportMessaging(),
+				edition,
+				countryCode,
+				modulesVersion: 'v1',
+				mvtId: Number(getCookie('GU_mvt_id')),
+			},
+		};
+		fetch(`${contributionsServiceUrl}/header`, {
+			method: 'post',
+			headers: { 'Content-Type': 'application/json' },
+			body: JSON.stringify(requestData),
+		})
+			.then((result) => result.json())
+			.then((response: { data?: SupportHeaderData }) => {
+				if (!response.data) {
+					return null;
+				}
+
+				setSupportHeaderResponse(response.data);
+				const { module, meta } = response.data;
+				return window
+					.guardianPolyfilledImport(module.url)
+					.then(
+						(headerModule: {
+							Header: React.FC<SupportHeaderProps>;
+						}) => {
+							setSupportHeader(() => headerModule.Header);
+							sendOphanComponentEvent(
+								'INSERT',
+								meta,
+								ophanRecord,
+							);
+						},
+					);
+			})
+			.catch((error) => {
+				// TODO - sentry
```

Comment:
we have a ticket to add sentry logs for epic/banner failures as well, so we'll do them together